test/database/skylink_test.go

```diff
@@ -165,8 +165,8 @@ func TestFindAndLock(t *testing.T) {
 
 	// Try to fetch an underpinned skylink, expect none to be found.
 	_, err = db.FindAndLockUnderpinned(ctx, cfg.ServerName, cfg.MinPinners)
-	if !errors.Contains(err, database.ErrSkylinkNotExist) {
-		t.Fatalf("Expected to get '%v', got '%v'", database.ErrSkylinkNotExist, err)
+	if !errors.Contains(err, database.ErrNoUnderpinnedSkylinks) {
```

Comment:
nit: don't you want to use your helper method here

Reply:
I wasn't going to `nit` on it but while scrolling through your MR I was going to remark that I think the helper is weirdly named, if you just follow the convention `isErrX` which then checks if the given error contains `errX` then this wouldn't have been confusing.

So I think renaming the helper to `IsErrNoUnderpinnedSkylinks` and using it here might be the F/U. But feel free to resolve it's a `nit`.

Reply:
Oh, sure, I'll rename it.